frame/democracy/src/lib.rs

```diff
@@ -240,16 +240,9 @@ enum Releases {
 
 #[frame_support::pallet]
 pub mod pallet {
-	use super::*;
-	use frame_support::{
-		dispatch::DispatchResultWithPostInfo,
-		pallet_prelude::*,
-		traits::EnsureOrigin,
-		weights::{DispatchClass, Pays},
-		Parameter,
-	};
-	use frame_system::{ensure_root, ensure_signed, pallet_prelude::*};
-	use sp_runtime::DispatchResult;
+	use super::{DispatchResult, *};
```

Comment:
Wouldn't this suffice?

```suggestion
	use super::*;
```

Reply:
I tried it, but compiler complained that

```
error[E0659]: `DispatchResult` is ambiguous (glob import vs glob import in the same module)
   --> frame/democracy/src/lib.rs:520:29
    |
520 |         Executed(ReferendumIndex, DispatchResult),
    |                                   ^^^^^^^^^^^^^^ ambiguous name
    |
note: `DispatchResult` could refer to the type alias imported here
   --> frame/democracy/src/lib.rs:243:9
    |
243 |     use super::*;
    |         ^^^^^^^^
    = help: consider adding an explicit import of `DispatchResult` to disambiguate
note: `DispatchResult` could also refer to the type alias imported here
   --> frame/democracy/src/lib.rs:245:6
    |
245 |     use frame_support::pallet_prelude::*;
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    = help: consider adding an explicit import of `DispatchResult` to disambiguate
```